bridge/src/headless-graphql-client.ts

```diff
@@ -31,7 +31,7 @@ export class HeadlessGraphQLClient implements IHeadlessGraphQLClient {
         return this._apiEndpoint;
     }
 
-    async getBlockIndex(blockHash: BlockHash): Promise<number> {
+    async getBlockIndex(blockHash: BlockHash): Promise<number> {        
```

Comment:
This change seems not intended.

```suggestion
    async getBlockIndex(blockHash: BlockHash): Promise<number> {
```

Reply:
udpated.